tradeActions.js

```diff
@@ -0,0 +1,36 @@
+async function getTradeData(url, actionName) {
+    const response = await fetch(url);
+    const jsonResponse = await response.json();
+    console.log(`======= ===== ===== ${actionName} ==== ==== ===== =====`);
+    console.log(jsonResponse);
+}
+
+function getKlines(symbol, interval = '1') {

```

Comment:
these functions should be async and wait for the call

Reply:
Done. made all the trading functions async